react-app/src/ExerciseOverview/ExerciseDetail.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { StyleSheet, Text, SafeAreaView, View, Pressable, FlatList } from 'react-native'
+import CancelIcon from '../../assets/icons/cancel.svg'
+import ThemeColor from '../Utilities/ThemeColor'
+import ExerciseImage from '../Utilities/ExerciseImage'
+
+export default function ExerciseDetail({ item, setShowDetail }) {
+  const { title, description, instruction, image } = item
+  return (
+    <SafeAreaView style={styles.container}>
+      <View style={styles.innerContainer}>
+        <Pressable style={styles.buttonContainer} onPress={() => setShowDetail(false)}>
+          <View style={styles.button}>
+            <CancelIcon />
+            <Text> Close </Text>
+          </View>
+        </Pressable>
+        <View style={styles.image}>
+          <ExerciseImage
+            image={image}
+            containerHeight={260}
+            containerWidth={380}
+            containerRadius={10}
+          />
+        </View>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.description}>{description}</Text>
+        <FlatList
+          data={instruction}
+          renderItem={({ item }) => <Text style={styles.instruction}>{item}</Text>}
```

Comment:
For long description, it should be scrollable otherwise user can't see everything